elements/pl-code/pl-code.mustache

```diff
@@ -1,12 +1,10 @@
 {{#prevent_select}}<!-- Well aren't you clever ;) -->{{/prevent_select}}
-<pre class="pl-code"><code
-{{#no_highlight}}class="nohighlight"{{/no_highlight}}
-{{#specify_language}}class="language-{{language}}"{{/specify_language}}
+<div class="pl-code"><span
 {{#prevent_select}}
 style="-moz-user-select: none; -webkit-user-select: none; -ms-user-select:none; user-select:none;-o-user-select:none;"
 unselectable="on"
 onselectstart="return false;"
 onmousedown="return false;"
 oncontextmenu="return false;"
 {{/prevent_select}}
->{{{code}}}</code></pre>
+>{{{code}}}</span></div>
```

Comment:
If we change `<code>` to `<span>`, do we also have to worry about things like wrapping and whitespace behavior being different by default?

Reply:
Don't think so, Pygments will automatically wrap its stuff with a `<pre>` tag so that's why I took it out, I think that's all we need to ensure formatting.

Reply:
Awesome! In that case the potential breakage for @gchallen is all we need to worry about.